tests/common/plugins/tacacs.py

```diff
@@ -1,8 +1,7 @@
 import pytest
 import crypt
 
-@pytest.fixture(scope="module")
-def setup_tacacs(ptfhost, duthost, creds):
+def start_tacacs(ptfhost, duthost, creds, tacacs_server_ip):
```

Comment:
i prefer the previous name, setup_tacacs. it means setup the tacacs on both client and server.

Reply:
Modified as per comment.